mithril-aggregator/src/commands/genesis_command.rs

```diff
@@ -229,3 +233,30 @@ impl BootstrapGenesisSubCommand {
         Ok(())
     }
 }
+
+/// Genesis keypair generation command.
+#[derive(Parser, Debug, Clone)]
+pub struct GenerateKeypairGenesisSubCommand {
+    /// Target path for the generated keypair
+    #[clap(long)]
+    target_path: PathBuf,
```

Comment:
Would it be useful to add a default path?

Reply:
I guess that expert users who will use this command need to know in advance where they want to store their keys (which is highly hazardous material for the protocol). This is why there is no default value for this parameter :+1: